### PR TITLE
Added Identifier to inverted snapshot tests

### LINKED ISSUE #51: Snapshotting with invert colors doesn't respect snapshot identifier
Specifically the Swift version of the snapshot method in the `iOSSnapshotTestCase` subspec. The Objective-C version of this behaves correctly.

To reproduce:
```swift
func testView() {
    let view = UIView(frame: .init(x: 0, y: 0, width: 100, height: 100))
    SnapshotVerifyWithInvertedColors(view, identifier: "someIdentifier")
}
```
When run in record mode, this should produce a reference image named `testView_someIdentifier_` followed by the device properties. Instead it produces a reference image named `testView_` (omitting the identifier).

### PR DESCRIPTION
**Pull request by joshmsamuels**
closes #51

### CODE REVIEW

**Comment by joshmsamuels** [comment]
@NickEntin here is a draft PR of my fix. The reason for this being a draft is the local inversion of the entire screen instead of just the colours

File: Example/AccessibilitySnapshot.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -48,6 +48,7 @@
 		3F8E5DB92535B25000276B32 /* SwiftUIView.swift in Sources */ = {isa = PBXBuildFile; fileRef = 3F8E5DB82535B25000276B32 /* SwiftUIView.swift */; };
 		3FEF854A2538455D0072611F /* SwiftUIView+EmbedInHostingController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 3FEF85492538455D0072611F /* SwiftUIView+EmbedInHostingController.swift */; };
 		3FEF854F253846420072611F /* SwiftUIViewWithScrollView.swift in Sources */ = {isa = PBXBuildFile; fileRef = 3FEF854E253846420072611F /* SwiftUIViewWithScrollView.swift */; };
+		56EBCA5525B8C42C0049F486 /* IdentifierTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 56EBCA5425B8C42C0049F486 /* IdentifierTests.swift */; };
```
Comment (joshmsamuels) [inline Example/AccessibilitySnapshot.xcodeproj/project.pbxproj:51]: This file should not be committed right? Should it be added to the gitignore?

File: Example/Gemfile.lock [hunk 2]
```diff
@@ -120,4 +120,4 @@ DEPENDENCIES
   xcodeproj!
 
 BUNDLED WITH
-   1.17.3
+   2.1.2
```
Comment (joshmsamuels) [inline Example/Gemfile.lock:123]: Should I leave this in?

**Comment by NickEntin** [comment]
@joshmsamuels I filed #56 to track fixing the flipped image issue

**Comment by joshmsamuels** [comment]
Thanks! @NickEntin sorry for another ping but do you have any idea why _some_ of the tests are failing? All the snapshots appear to be the same and all the tests pass locally.

It seems really odd that the iOS 12 tests are passing here but 13 & 14 are failing

**Comment by NickEntin** [comment]
> do you have any idea why some of the tests are failing?

@joshmsamuels I'm having trouble reproducing the issue locally. See my comment in #56. Can you test locally and see which path it's taking?

**Pull request closed without merging**